src/controller/events.ts

```diff
@@ -79,6 +79,7 @@ const CommandsLookup: {[s: string]: MessagePayloadType} = {
     'getPinCodeRsp': 'commandGetPinCodeRsp',
     'arrivalSensorNotify': 'commandArrivalSensorNotify',
     'getPanelStatus': 'commandGetPanelStatus',
+    'checkin': 'commandCheckin',
```

Comment:
Should probably be: `'checkIn': 'commandCheckIn',`

Based on https://www.merriam-webster.com/dictionary/check-in

Reply:
`CheckIn` seems better indeed, thanks